db/re/skill_db.conf

```diff
@@ -26754,18 +26743,7 @@ skill_db: (
 			Lv9: 120
 			Lv10: 130
 		}
-		SpiritSphereCost: {
-			Lv1: 1
-			Lv2: 2
-			Lv3: 3
-			Lv4: 4
-			Lv5: 5
-			Lv6: 6
-			Lv7: 7
-			Lv8: 8
-			Lv9: 9
-			Lv10: 10
-		}
+		SpiritSphereCost: 2
```

Comment:
Are you sure this isn't iRO-only?
According to the skill description in kRO and twRO only 1 sphere is required/consumed...

Reply:
I just tested `Gentle Touch Revitalize` in my twRO zone and it requires/consumes 1 sphere per used skill level.
Well, that zone is EP 14 so maybe the requirement has changed until today. :confused:

Reply:
I will do more test on kRO this november...

Reply:
so after checking, it does still consumes 1 sphere per level.